wallets/provider-walletconnect-2/src/index.ts

```diff
@@ -211,7 +209,7 @@ export const disconnect: Disconnect = async ({ instance }) => {
   const { client } = instance as WCInstance;
 
   if (client) {
-    void disconnectSessions(client);
+    void disconnectSessions(client).catch((error) => console.log(error));
```

Comment:
I guess you should remove `.catch` as well.

Reply:
I think it's not correct. Because the wallet is disconnected but we sometimes raise errors in disconnect because of try to disconnect all previous pairings session s in disconnect flow which some of them don't exist and it's not logical to raise error for them.

Reply:
That's fair. Could you please use `logError` instead?

Reply:
Why we should report this to sentry? I didn't understand.